apps/api/src/bootstrap.ts

```diff
@@ -45,9 +45,12 @@ export async function bootstrap(expressApp?): Promise<INestApplication> {
 
   let app: INestApplication;
   if (expressApp) {
-    app = await NestFactory.create(AppModule, new ExpressAdapter(expressApp));
+    app = await NestFactory.create(AppModule, new ExpressAdapter(expressApp), {
+      bodyParser: false,
+      rawBody: true,
+    });
```

Comment:
I'm wondering shouldn't we only do this when there is a billing module? Like:
```
if (hasBillingModule) {
   app = await NestFactory.create(AppModule, new ExpressAdapter(expressApp), {
      bodyParser: false,
      rawBody: true,
    });
    const rawBodyBuffer = require('@novu/ee-billing')?.rawBodyBuffer;
    app.use(bodyParser.json({ verify: rawBodyBuffer }));
    app.use(bodyParser.urlencoded({ extended: true, verify: rawBodyBuffer }));
} else {
   app = await NestFactory.create(AppModule, new ExpressAdapter(expressApp));
}

```
of course in a nicer way ;) because I'm worried whether this will still work for OS

Reply:
This condition is now handled via the `NOVU_ENTERPRISE=true` flag below, however the presence of `bodyParser:false , rawBody: true` is tightly coupled to the code block below.

I prefer the approach Paweł has suggested, to keep the coupled functionality under the same conditional block. If we use the `NOVU_ENTERPRISE=true` flag for the condition instead of `hasBillingModule`, we get the best of both.

Reply:
I have now updated my code please check if that was something similar to what you have in mind :)